Add a comfyui custom node for Imagen4 (#196)
GoogleCloudPlatform/accelerated-platforms: platforms/gke/base/use-cases/inference-ref-arch/examples/comfyui/README.md

## ComfyUI workflows

> This guide provides six workflows out of the box. On ComfyUI, click on

Reviewer: ```suggestion
This guide provides example workflows out of the box. On ComfyUI, click on
```

> WORKFLOWS. You will six json workflow files.

Reviewer: ```suggestion
WORKFLOWS. You should see the following workflow files:
```

Click any of the workflow files to open the workflow and click `Run` to see the
results.

> Additionally, you can create more workflows using Imagen and Veo custom nodes.

Reviewer: ```suggestion
You can create additional workflows using the Imagen and Veo custom nodes.
```

> To view these custom nodes, right click in ComfyUI screen, choose `Google AI` in

Reviewer: ```suggestion
To view these custom nodes, right click in the ComfyUI screen, choose `Google AI` in
```
